hedy.py

```diff
@@ -174,6 +174,17 @@ def needs_colon(rule):
     return f'{rule[0:pos]} _COLON {rule[pos:]}'
 
 
+def translate_index_error(code, liste_name):
```

Comment:
If it's a util method only used internally in `hedy.py` it's standard python practice to add an underscore to the method. So `_translate_index_error` . That signals to the developers that it's a one-of util method, but python doesn't treat it any different.